src/util/schemas/Validator.ts

```diff
@@ -43,7 +43,7 @@ export const ajv = new Ajv({
 	allowUnionTypes: true,
 });
 
-addFormats(ajv);
+addFormats(ajv as never);
```

Comment:
this is needed when installing with yarn, because apparently ajv-formats brings it own types which are not compatible with ajv